runtime/compiler/trj9/runtime/J9Profiler.hpp

```diff
@@ -91,7 +104,8 @@ class TR_PersistentProfileInfo
         _catchBlockProfileInfo(NULL),
         _blockFrequencyInfo(NULL),
         _valueProfileInfo(NULL),
-        _callSiteInfo(NULL)
+        _callSiteInfo(NULL),
+        _refCount(1)
```

Comment:
You should explicitly document what can increment the ref counts. For example, in the constructor of `TR_PersistentProfileInfo`, the refcount is set to 1 because the `TR_PersistentBodyInfo` creates a `TR_PersistentProfileInfo` and hence has a reference to it. The other three scenarios I can think of are:

* _recentProfileInfo in `TR_PersistentMethodInfo`
* _bestProfileInfo in `TR_PersistentMethodInfo`
* Some compilation thread via `TR_AccessedProfileInfo`

Reply:
I have extended the comment above to include the additional details.